docs/user_guide/models/mrvi.md

```diff
@@ -0,0 +1,146 @@
+# MrVI
+
+**MrVI** [^ref1] (Multi-resolution Variational Inference; Python class
+{class}`~scvi.external.MRVI`) is a deep generative model designed for the analysis of large-scale
+single-cell transcriptomics data with multi-sample, multi-batch experimental designs.
+
+The advantages of MrVI are:
+
+- Facilitates both exploratory analysis to divide samples into groups based on
+    molecular properties and comparative analysis to compare pre-defined groups of samples.
+- Provides single-cell-resolution differential expression and differential abundance estimates.
+- Captures nonlinear, cell-type specific effects of sample-level covariates on gene expression.
+- Scales to large datasets with millions of cells and hundreds of samples.
+
```

Comment:
```suggestion
MrVI conducts both **exploratory analyses** (locally dividing samples into groups based on molecular properties)
and **comparative analyses** (comparing pre-defined groups of samples in terms of differential expression and differential abundance) at single-cell resolution.
It can capture nonlinear and cell-type specific variation of sample-level covariates on gene expression.

For this purpose, MrVI requires to specify (1) a sample-level target covariate of interest (that generally corresponds to sample IDs) and (2) nuisance covariates (e.g., sequencing batch, processing day) that introduce technical variation.

```